code/modules/mob/living/carbon/monkey/monkey.dm

```diff
@@ -157,9 +157,6 @@
 		return 0
 	return 1
 
-/mob/living/carbon/monkey/can_use_guns(obj/item/G)

```

Comment:
This will prevent monkeys from using guns at all since the living check checks for AdvancedToolUser